front/app/containers/IdeasNewPage/IdeasNewSurveyForm/SurveyHeading.tsx

```diff
@@ -31,31 +33,17 @@ const StyledSurveyTitle = styled(Text)`
 `;
 
 type Props = {
-  project: IProjectData;
   titleText: string | React.ReactNode;
-  canUserEditProject: boolean;
-  loggedIn?: boolean;
-  percentageAnswered: number;
 };
 
-const SurveyHeading = ({
-  project,
-  titleText,
-  canUserEditProject,
-  loggedIn,
-  percentageAnswered,
-}: Props) => {
-  const theme = useTheme();
+const SurveyHeading = ({ titleText }: Props) => {
+  const { slug: projectSlug } = useParams();
+  const { data: project } = useProjectBySlug(projectSlug);
+  const { data: authUser } = useAuthUser();
+
   const { formatMessage } = useIntl();
   const [searchParams] = useSearchParams();
-  const phaseId =
-    searchParams.get('phase_id') ||
-    project.relationships.current_phase?.data?.id;
-  const linkToSurveyBuilder: RouteType = `/admin/projects/${project.id}/phases/${phaseId}/native-survey/edit`;
-  const canEditSurvey = canUserEditProject;
   const isSmallerThanPhone = useBreakpoint('phone');
-  const showEditSurveyButton = !isSmallerThanPhone && canEditSurvey;
```

Comment:
Instead of passing the canUserEditProject prop we can just use canModerate directly. Also no need to create all these variables. See showEditSurveyButton on the right:

```
const showEditSurveyButton = !isSmallerThanPhone && canModerateProject(project.data.id, authUser);
```